1_fetch/src/functions.R

```diff
@@ -0,0 +1,15 @@
+fetch <- function(sb_id){
```

Comment:
Good. I also recommend pulling out the `project_output_dir` from the body of the function and making it an argument.

Reply:
Thanks!